Tests/Data/ThermoHydroMechanics/Linear/Beam_sealed_bimaterial/square_1e2.prj

```diff
@@ -0,0 +1,317 @@
+            <solid_specific_heat_capacity>C_s</solid_specific_heat_capacity>
+            <solid_thermal_conductivity>lambda_s</solid_thermal_conductivity>
+            <fluid_specific_heat_capacity>C_f</fluid_specific_heat_capacity>
+            <fluid_thermal_conductivity>lambda_f</fluid_thermal_conductivity>
+            <reference_temperature>T0</reference_temperature>
+            <process_variables>
+                <displacement>displacement</displacement>
+                <pressure>pressure</pressure>
+                <temperature>temperature</temperature>
+            </process_variables>
+            <secondary_variables>
+                <secondary_variable type="static" internal_name="sigma" output_name="sigma"/>
+				<secondary_variable type="static" internal_name="epsilon" output_name="epsilon"/>
+             </secondary_variables>
+            <specific_body_force>0 0</specific_body_force>
+        </process>
+    </processes>
+    <time_loop>
+        <processes>
+            <process ref="THERMO_HYDRO_MECHANICS">
+                <nonlinear_solver>basic_newton</nonlinear_solver>
+                <convergence_criterion>
+                    <type>Residual</type>
+                    <norm_type>NORM2</norm_type>
+                    <abstol>1e-5</abstol>
+                </convergence_criterion>
+                <time_discretization><type>BackwardEuler</type></time_discretization>
+                <time_stepping>
+                    <type>FixedTimeStepping</type>
+                    <t_initial>0</t_initial>
+                    <t_end>100</t_end>
+                    <timesteps>
+                        <pair><repeat>100</repeat>
+                        <delta_t>10</delta_t></pair>
+                    </timesteps>
+                </time_stepping>
+            </process>
+        </processes>
+        <output>
+            <type>VTK</type>
+            <prefix>square_1e2</prefix>
+            <timesteps>
+                <pair><repeat>1</repeat>
+                <each_steps>10</each_steps></pair>
+            </timesteps>
+                <variables>
+                    <variable>displacement</variable>
+                    <variable>pressure</variable>
+                    <variable>temperature</variable>
+                    <variable>sigma</variable>
+                    <variable>epsilon</variable>
+                </variables>
+        </output>
+    </time_loop>
+    <parameters>
+        <parameter>
+            <name>E</name>
+            <type>Group</type>
+            <group_id_property>MaterialIDs</group_id_property>
+            <index_values>
+            <index>0</index>
+            <value>19</value>
+            </index_values>
+            <index_values>
+            <index>1</index>
+            <value>0.95</value>
+            </index_values>
+            <index_values>
+            <index>2</index>
+            <value>9.5</value>
```

Comment:
This value is not mentioned in the documentation. Maybe I didn't see it. A table of input parameters would be nice in the documentation.

Reply:
This value is still not in the documentation.

Reply:
The parameters are added in the documentation.